crates/ruff/src/rules/flake8_bugbear/rules/mutable_argument_default.rs

```diff
@@ -175,7 +175,7 @@ fn move_initialization(
                 return None;
             }
             Edit::insertion(content, locator.line_start(statement.start()))
-        } else if statement.end() == locator.text_len() {
+        } else if locator.full_line_end(statement.end()) == locator.text_len() {
```

Comment:
Do we care about the cost of calculating the `full_line_end` twice here? (we do it on L183 as well)